### PR TITLE
LPS-190006 Replace aui:button with clay:button in Fragment

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-190006)

## Steps

**Case 1: fragment-web**
1. Go to Control Menu > Control Panel > System Settings
2. Under Content & Data > Page Fragments
Screenshot 2023-07-31 at 15 14 12

**Case 2: fragment-collection-filter-category**

1. Go to Site Builder > Collections 
2. Create a Manual Collection > Item type > All types 
3. Go to Site Builder > Pages > Create a Content Page
4. Add a Collection Display Fragment and a Collection Filter Fragment
5. Map Collection Display Fragment to the collection 
6. In Collection Filter > select the collection > Filter by Category
Screenshot 2023-08-03 at 16 18 21

**Case 3: fragment-renderer-collection-filter-impl**
1. Go to Site Builder > Collections 
2. Create a Manual Collection > Item type > All types 
3. Go to Site Builder > Pages > Create a Content Page
4. Add a Collection Display Fragment, a Collection Filter Fragment and a Applied Filters Fragment
5. Map Collection Display Fragment to the collection 
6. In Collection Filter > select the collection > Filter by Keywords
7. In Applied Filters, map it to the collection created
8. Duplicate 3 Fragments of Applied Filters Fragment 
9. Go to the page content view 
10. Type long characters (such as "asjdkjasdkjhasdhaoishdoahsdhasoudhasouhdasjdkjasdkjhasdhaoishdoahsdhasoudhasouhdasjdkjasdkjhasdhaoishdoahsdhasoudhasouhdasjdkjasdkjhasdhaoishdoahsdhasoudhasouhd") in the Collection Filter search bar and see the applied filters below 
Screenshot 2023-08-04 at 18 51 01

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/fragment_service_configuration.jsp [hunk 1]
```diff
@@ -53,9 +53,22 @@ FragmentServiceConfigurationDisplayContext fragmentServiceConfigurationDisplayCo
...
 		<clay:sheet-footer>
-			<aui:button primary="<%= true %>" type="submit" value="save" />
+			<clay:button
+				displayType="primary"
+				label="save"
+				type="submit"
+			/>
 
-			<aui:button href="<%= fragmentServiceConfigurationDisplayContext.getRedirect() %>" type="cancel" />
+			<clay:button
```
Comment (sandrodw3) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/fragment_service_configuration.jsp:62]: Why not a `clay:link` here?

**Comment by BarbaraCabrera** [comment]
Ey, @DiegoHu97 ! 
I was checking the buttons that need to be changed in the fragment module and I saw this ones:

https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/fragment/fragment-collection-filter-category/src/main/resources/META-INF/resources/page.jsp#L16-L17

https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/resources/META-INF/resources/applied_filters.jsp#L54

Would you mind take a look at them and see if we can change them too? So instead of replace the buttons only in fragment-web we replace everything in the fragment module. 

Thanks in advance! 😄

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
@DiegoHu97 just the disabled attribute I mentioned to you!

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**